src/framework/uicomponents/qml/MuseScore/UiComponents/FlatButton.qml

```diff
@@ -33,30 +33,10 @@ FocusableControl {
 
     opacity: root.enabled ? 1.0 : ui.theme.itemOpacityDisabled
 
-    onInternalClicked: {
-        root.clicked()
-    }
-
-    onInternalPressAndHold: {
-        root.pressAndHold()
-    }
-
-    onInternalTriggered: {
-        root.clicked()
-    }
+    mouseArea.onClicked: root.clicked()
+    mouseArea.onPressAndHold: root.pressAndHold()
 
-    mouseArea.hoverEnabled: true
-    mouseArea.onContainsMouseChanged: {
-        if (!Boolean(root.hint)) {
-            return
-        }
-
-        if (mouseArea.containsMouse) {
-            ui.tooltip.show(this, root.hint)
```

Comment:
do we no longer show the tooltip for buttons on hover?

Reply:
Oops, this is a mistake, possibly an incorrectly resolved conflict. 
Fixed (returned), thanks!